mautic/mautic
# Replaced %contacts% to %leads%

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed issues  |  N/A

## Description
~~A few %leads% occurrences were missed while renaming to %contacts% and so some contact counts does not display.~~

All occurrences of %contacts% and replaced them back to %leads% to keep BC for translations.

## Steps to reproduce the bug (if applicable)
Try to run `app/console mautic:campaigns:update`. The output does not display the contact count like this:

> %contacts% total contact(s) to be added in batches of 300

## Steps to test this PR

Apply this PR, clear the cache and run the command again. It should display the contact count like before renaming:

> 0 total contact(s) to be added in batches of 300
